### PR TITLE
[SLP][NFC] Add a new test for store chains that mix different sizes of values

### PR DESCRIPTION
**Pull request by tcwzxx**

### CODE REVIEW

File: llvm/test/Transforms/SLPVectorizer/X86/stores_mix_sizes.ll [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+; RUN: opt -S --passes=slp-vectorizer -mtriple=x86_64-unknown-linux < %s | FileCheck %s
+
+define void @test(ptr %p) {
+; CHECK-LABEL: define void @test(
+; CHECK-SAME: ptr [[P:%.*]]) {
+; CHECK-NEXT:  [[ENTRY:.*:]]
+; CHECK-NEXT:    [[IDX1:%.*]] = getelementptr i8, ptr [[P]], i64 1
+; CHECK-NEXT:    [[IDX_64_9:%.*]] = getelementptr i64, ptr [[P]], i64 9
+; CHECK-NEXT:    store i64 1, ptr [[IDX_64_9]], align 8
+; CHECK-NEXT:    store <8 x i8> zeroinitializer, ptr [[IDX1]], align 4
```
Comment (alexey-bataev) [inline llvm/test/Transforms/SLPVectorizer/X86/stores_mix_sizes.ll:11]: The check must be formed with the current version of the compiler. If it is vectorized already, your patch does nothing.

Comment (tcwzxx) [reply llvm/test/Transforms/SLPVectorizer/X86/stores_mix_sizes.ll:11]: The check is generated after the patch, so the correct process is to add a current version of the compiler test case (NFC) and then update this test case in the patch?

Comment (alexey-bataev) [reply llvm/test/Transforms/SLPVectorizer/X86/stores_mix_sizes.ll:11]: Yes

Comment (tcwzxx) [reply llvm/test/Transforms/SLPVectorizer/X86/stores_mix_sizes.ll:11]: Thanks

**Review by alexey-bataev — APPROVED** [review]
LG

**Comment by tcwzxx** [comment]
> LG

Could you help me merge it?

**Pull request merged**